src/routes/analytics/DashboardPage.tsx

```diff
@@ -10,15 +10,12 @@ interface Props {
 export function DashboardPage(props: Props) {
   const dashboardId = parseInt(props.id, 10)
 
+  // When the site changed, reload to apply the site's sandboxed data.
+  useSiteChanged(window.location.reload)
```

Comment:
Without this, the dashboard shows sandboxed data from the old user, as there is no way for the SDK to reload the current user.